model/compile/QtiAssetCompiler/QtiItemAssetXmlReplacer.php

```diff
@@ -47,6 +48,30 @@ public function replaceAssetNodeValue(DOMDocument $domDocument, array $packedAss
             }
         }
 
+        $this->replaceEncodedNodes($xpath, $packedAssets);
+
         return $packedAssets;
     }
+
+    /**
+     * Replace assets in html encoded properties (such as content of text reader interaction)
+     * @param $xpath
+     * @param array $packedAssets
+     */
+    private function replaceEncodedNodes($xpath, array $packedAssets)
```

Comment:
because this is a new piece of code,  i would like to ask you to 
 - provide the strong type for the xpath
 - add return type
remove the phpfoc as it does not provide useful information